spacebar

```
@@ -3,214 +3,189 @@
         ;;
     logs)
-        if [[ -z "$service_name" ]]; then 
-            echo "Must specify a service with --service (client|server)."
-         else
-            echo "Streaming most recent logs for services matching: $service_name"
-         if [[ "$service_name" = "client" ]]; then
-                echo "Setting client-level logs to debug"
-                heroku logs --app spacebar --tail
-         elif [[ "$service_name" = "server" ]]; then
-                heroku logs --app spacebar-5 --tail
-         else
-                echo "Invalid service $service_name, must be one of (client|server)."
-         fi
-    fi
-    ;;
+        [[ -z "$service_name" ]] && die "Specify a service with --service (client|server|all)."
+        handle_service_operation heroku_logs
+        ;;
     deploy)
-        if [[ -z "$service_name" ]]; then
-            echo "Must specify a service with --service (client|server)."
-        else
-            echo "Deploying: $service_name"
-            if [[ "$service_name" = "client" ]]; then
-								increment_version "spacebar"
-								. $ENV_LOC # Source to update the version in the current shell.
-								release_to_sentry "$service_name"
-                heroku builds:create -a spacebar
-            elif [[ "$service_name" = "server" ]]; then
-                cd server
-								increment_version "spacebar-5"
-								. $ENV_LOC # Source to update the version in the current shell.
-								release_to_sentry "$service_name"
-								heroku builds:create -a spacebar-5 
-								cd ..
-            elif [[ "$service_name" = "all" ]]; then
-                ./spacebar deploy --service server; ./spacebar deploy --service client
-            else
-                echo "Invalid service $service_name, must be one of (client|server|all)."
-            fi
-        fi
+        [[ -z "$service_name" ]] && die "Specify a service with --service (client|server|all)."
```

not sure if important, but these will all fail silently if you pass in `-s` without an argument

---

really good catch -- fixed